v0/destinations/pinterest_tag/transform.js

```diff
@@ -159,9 +116,65 @@ const processEcomFields = (message, mandatoryPayload) => {
   };
 };
 
+const commonFieldResponseBuilder = (
+  message,
+  { Config },
+  messageType,
+  eventName
+) => {
+  let processedUserPayload;
+  const { appId, advertiserId, deduplicationKey, sendingUnHashedData } = Config;
+  // ref: https://s.pinimg.com/ct/docs/conversions_api/dist/v3.html
+  const processedCommonPayload = processCommonPayload(message);
+  /*
+    message deduplication facility is provided *only* for the users who are using the *new configuration*.
+    if  "enableDeduplication" is set to *true* and "deduplicationKey" is set via webapp, that key value will be
+    sent as "event_id". On it's absence it will fallback to "messageId".
+    And if "enableDeduplication" is set to false, it will fallback to "messageId"
+  */
```

Comment:
These can be moved as part of this function's JS doc

Reply:
moved